cve_bin_tool/cli.py

```diff
@@ -255,6 +259,12 @@ def main(argv=None):
         with ErrorHandler(logger=LOGGER, mode=ErrorMode.NoTrace):
             raise FileNotFoundError("Directory/File doesn't exist")
 
+    for exclude_path in args["exclude"].split(","):
+        if exclude_path and not os.path.exists(exclude_path):
+            parser.print_usage()
+            with ErrorHandler(logger=LOGGER, mode=ErrorMode.NoTrace):
+                raise FileNotFoundError("Exclude directory doesn't exist")
+
```

Comment:
You don't need to do this.  Just use StringToListAction which will convert string to list automatically.

Reply:
Also we shouldn't terminate program if exclude_path does not exists. User may have common config file to scan bunch of directories and some of them may not contain exclude_path of other.

Reply:
> Also we shouldn't terminate program if exclude_path does not exists. User may have common config file to scan bunch of directories and some of them may not contain exclude_path of other.

Should I throw a warning if that exclude path doesn't exist?

Reply:
I don't think that's necessary but @terriko can give better suggestion on this.